### PR TITLE
Incorrect z-index fixed

### PR DESCRIPTION
**Pull request by icy-meteor**

### CODE REVIEW

**Review by daltonfury42 — APPROVED** [review]
I think this makes sense. @maaverik can you take a look?

**Review by daltonfury42 — CHANGES_REQUESTED** [review]
The text becomes unclickable. Can you take a look?

**Comment by icy-meteor** [comment]
Sure, taking a look.

**Comment by icy-meteor** [comment]
Looks like the way navbar elements are laid down is causing some overlaps in margin. I am refactoring the code in a better and cleaner way @daltonfury42

**Comment by icy-meteor** [comment]
Fixed it! Can you take a look now @daltonfury42  :)

**Comment by maaverik** [comment]
Looks like this works now. @daltonfury42 could you also check?

File: simplq/src/components/pages/Home/Nav/LeftNav.jsx [hunk 1]
```diff
@@ -12,24 +12,26 @@ const scrollToHowItWorks = () => {
...
+          <a>My Queues</a>
+        </li>
+        <li>
+          <a tabIndex={-1} href="https://iimb.qualtrics.com/jfe/form/SV_aY8FY91ztRX9NvD">
+            Contact Us
+          </a>
+        </li>
+        <li>
+          <a>Sign In/Sign Up</a>
+        </li>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/Nav/LeftNav.jsx:32]: Is there any change here? Or was it just removing the left in the scss file that fixed this?

Comment (icy-meteor) [reply simplq/src/components/pages/Home/Nav/LeftNav.jsx:32]: Also, wrapped the entire <ul> under a <div> tag in LeftNav.jsx component to get it working :)

**Comment by daltonfury42** [comment]
I've generated a preview here: https://click.dpmhbfuiok7f8.amplifyapp.com/

**Review by daltonfury42 — APPROVED** [review]
I can confirm that the requirements are met.

File: simplq/src/components/pages/Home/Nav/LeftNav.jsx (hunk 1 above)
Comment (daltonfury42) [reply simplq/src/components/pages/Home/Nav/LeftNav.jsx:32]: +1 How did this fix the isssue?

Comment (icy-meteor) [reply simplq/src/components/pages/Home/Nav/LeftNav.jsx:32]: There are multiple ways to fix this. Ideally, it is something to do with the way CSS positioning has been used. Wrapping in a div, would ensure that anything around the div is consuming its entire width and height, giving space for other div's as well. In this case, the `<ul>` was not wrapped in a div and the browser couldn't calculate exact width consumed by it, hence the left property pulled it on top of the logo text :)
I know it sounds confusing, but hope it helps.

Comment (maaverik) [reply simplq/src/components/pages/Home/Nav/LeftNav.jsx:32]: Okay, I didn't know that. Thanks.

**Pull request merged**